src/Process/3dTilesProcessing.js

```diff
@@ -18,6 +18,11 @@ function requestNewTile(view, scheduler, geometryLayer, metadata, parent, redraw
         redraw,
     };
 
+    if (geometryLayer.source.isC3DTilesGoogleSource) {
+        // special case to handle non-standard 3dTilesGoogle behavior
```

Comment:
can you link the issue on the 3D Tiles repo about this special behaviour please?

Reply:
done